plugin.video.embycon/resources/lib/downloadutils.py

```diff
@@ -313,7 +344,7 @@ def getArtwork(self, data, art_type, parent=False, index=0, server=None):
             log.debug("No Image Tag for request:{0} item:{1} parent:{2}", art_type, item_type, parent)
             return ""
 
-        artwork = "%s/emby/Items/%s/Images/%s/%s?Format=original&Tag=%s" % (server, id, art_type, index, imageTag)
+        artwork = "%s/emby/Items/%s/Images/%s/%s?Format=original&Tag=%s|verifypeer=false" % (server, id, art_type, index, imageTag)
```

Comment:
`verifypeer` should not be disabled for all users. Make it a setting with default to verifypeer enabled.

Reply:
I will have a look at this but not sure I can, I spent a bunch of time but the function is used in a lot of places and it get the setting it is expensive and reduced performance too much.

Reply:
also the check is still done on the API requests so it wont even get to this stage unless either the SLL cert is verifiable or the user set to ignore cert states.

Reply:
use the verify_cert setting to append the verifypeer=false is needed